src/traces/sunburst/attributes.js

```diff
@@ -0,0 +1,145 @@
+/**
+* Copyright 2012-2019, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+'use strict';
+
+var plotAttrs = require('../../plots/attributes');
+var hovertemplateAttrs = require('../../components/fx/hovertemplate_attributes');
+var domainAttrs = require('../../plots/domain').attributes;
+var pieAtts = require('../pie/attributes');
+
+var extendFlat = require('../../lib/extend').extendFlat;
+
+// TODO should we use singular `label`, `parent` and `value`?
+
+module.exports = {
+    labels: {
+        valType: 'data_array',
+        editType: 'calc',
+        description: [
+            'Sets the labels of each of the sunburst sectors.'
+        ].join(' ')
+    },
+    parents: {
+        valType: 'data_array',
+        editType: 'calc',
+        description: [
+            'Sets the parent sectors for each of the sunburst sectors.',
+            'Empty string items \'\' are understood to reference',
+            'the root node in the hierarchy.',
+            'If `ids` is filled, `parents` items are understood to be "ids" themselves.',
+            'When `ids` is not set, plotly attempts to find matching items in `labels`,',
+            'but beware there must be unique.'
+        ].join(' ')
+    },
+
+    values: {
+        valType: 'data_array',
+        editType: 'calc',
+        description: [
+            'Sets the values associated with each of the sunburst sectors.',
+            'Use with `branchvalues` to determine how the values are summed.'
+        ].join(' ')
+    },
+    branchvalues: {
+        valType: 'enumerated',
+        values: ['total', 'extra'],
```

Comment:
rename to `'remainder'`

Reply:
done in https://github.com/plotly/plotly.js/pull/3594/commits/1258f2da606954952b7e2bfc1c82eb7a274b2ef1